lib/league_stats.rb

```diff
@@ -19,6 +19,16 @@ def worst_offense
     find_name(worst_offense_id)
   end
 
+  def best_defense
+    best_defense_id = defense_helper.max_by { |id, goals| goals }.first
+    find_name(best_defense_id)
+  end
+
+  def worst_defense
+    worst_defense_id = defense_helper.min_by { |id, goals| goals }.first
+    find_name(worst_defense_id)
+  end
+
```

Comment:
Just moved these two methods up a few lines to match the order they're shown in project specs

Reply:
I really like how you've organized the methods and tests to match up with the specs.

Reply:
^^^ 
One of the best if not best project I've worked on with how organized this is. Super easy to follow and build off of. Good work Alex!